CRM/Core/Payment/Form.php

```diff
@@ -360,6 +360,15 @@ public static function validateCreditCard($values, &$errors, $processorID = NULL
    * @param bool $reverse
    */
   public static function mapParams($id, $src, &$dst, $reverse = FALSE) {
+    // Set text version of state & country if present.
```

Comment:
hmm now I'm a bit torn - do I introduce more risk if I remove them now? Probably not since @jitendrapurohit was ready to remove them before. Will wait on Jitendra's opinion